org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -74,13 +76,15 @@ public void testCoverageResult() {
 				ICounter.FULLY_COVERED);
 		assertLine("explicitExceptionTryCatch.before", ICounter.FULLY_COVERED);
 		assertLine("explicitExceptionTryCatch.throw", ICounter.FULLY_COVERED);
+		assertLine("explicitExceptionTryCatch.catch", ICounter.FULLY_COVERED);
 		assertLine("explicitExceptionTryCatch.catchBlock",
 				ICounter.FULLY_COVERED);
 
 		// 7. Finally Block Without Exception Thrown
 		// Finally block is yellow as the exception path is missing.
 		assertLine("noExceptionFinally.beforeBlock", ICounter.FULLY_COVERED);
 		assertLine("noExceptionFinally.tryBlock", ICounter.FULLY_COVERED);
+		assertLine("noExceptionFinally.finally", ICounter.EMPTY);
```

Comment:
With Eclipse compiler:

    Status in line 116: 		} finally { // $line-noExceptionFinally.finally$ expected:<[EMPTY]> but was:<[PARTLY_COVERED]>